assets/css/inloop.css

```
@@ -72,6 +72,18 @@ code {
   color: #ddd;
 }
 
+/*
+ * Insert an invisible spacer on selected headings
+ * to force them below the sticky tab bar
+ */
+.static-tab-content :target:before {
+  content: '';
+  display: block;
+  width: 0;
+  height: 5em;
+  margin-top: -5em;
```

```suggestion
  height: 65px;
  margin-top: -65px;
```
The original 5em value is a bit too large in my opinion. I suggest using the total height of the tab bar plus 10px, and round it up to a nice value such as 65px.